src/fairseq2/recipes/llama/convert_checkpoint.py

```diff
@@ -153,8 +153,19 @@ def __call__(self, args: Namespace, resolver: DependencyResolver) -> None:
             if model_config.num_attn_heads != model_config.num_key_value_heads:
                 params["model"]["n_kv_heads"] = model_config.num_key_value_heads
 
-            if args.arch == "llama2_70b" or args.arch.startswith("llama3"):
-                params["model"]["ffn_dim_multiplier"] = 1.3
+            # we only specify archs where multiplier != 1.0
+            ffn_dim_multipliers = {
+                "llama2_70b": 1.3,
+                "llama3_8b": 1.3,
+                "llama3_70b": 1.3,
+                "llama3_1_8b": 1.3,
+                "llama3_1_70b": 1.3,
+                "llama3_1_405b": 1.2,
+                "llama3_2_1b": 1.5,
+            }
+
+            if args.arch in ffn_dim_multipliers:
```

Comment:
Not necessarily for this PR, but should we print a warning message if the architecture name is not known? The quickest way would be to do some basic string/regex check. A more sophisticated way would be to check the LLaMA ConfigRegistry.